hc-nested-list-behavior.html

```diff
@@ -62,6 +62,28 @@
       'item-open-changed': 'itemOpenChanged'
     },
     /**
+     * will recursively descend down the tree closing each expanded tree node.
+     * Relies upon logic being in place to prevent two sibling nodes from being open concurrently.
+     *
+     * @param {Object} [parts = this.parts] - object containing all properties of each node
+     * @param {Array.<Object>} [items = this.parts.root] - root Object to begin closing nodes from.
+     * @return {Boolean} - returns true only if it sees a node with no children or all its child
+     *                     nodes are closed.
+     */
+    collapseTree(parts = this.parts, items = this.parts.root) {
+      if (items.some(item => item.open === false)) return;
+      items.some((item) => {
+        if (item.open) item.open = false;
+        if (parts[item.id]) {
+          if (item.hasChildren = false || parts[item.id].every(subItem => !subItem.open))
+            return true;
+        } else if (item.hasChildren = false) {
+          return true;
+        }
+        this.collapseTree(parts, parts[item.id]);
```

Comment:
This is calling on every item up to the point of the one that is currently open, also unnecessary since only the one that is open and it's children matter.